nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/HandleHttpResponse.java

```diff
@@ -169,6 +174,7 @@ public void onTrigger(final ProcessContext context, final ProcessSession session
             return;
         }
 
+        session.getProvenanceReporter().send(flowFile, flowFile.getAttribute(HTTP_REQUEST_URI), stopWatch.getElapsed(TimeUnit.MILLISECONDS));
```

Comment:
The Request URI would be the URI of NiFi, I believe. What we really should be including here is the URI where the data was sent. So I think we would need to build that up by using ( "http://" + flowFile.getAttribute(HTTP_REMOTE_ADDR) ) or something like that? To be more accurate, rather than using http:// we should probably use either 'http' or 'https' depending on what is present in the HTTP_REQUEST_URI. Thoughts?